app/perfunwind.cpp

```diff
@@ -115,7 +115,11 @@ static bool memoryRead(Dwfl *dwfl, Dwarf_Addr addr, Dwarf_Word *result, void *ar
 
     /* Check overflow. */
     if (addr + sizeof(Dwarf_Word) < addr) {
-        qDebug() << "Invalid memory read requested by dwfl" << Qt::hex << addr;
+        static Dwarf_Addr last_invalid_addr;
```

Comment:
don't use a static, make it a member function local to the `UnwindInfo` struct above

Reply:
oh and use camelCase  instead of snake_case